Libraries/LibGUI/ColumnsView.cpp

```diff
@@ -62,6 +62,28 @@ ColumnsView::~ColumnsView()
 {
 }
 
+void ColumnsView::select_all()
+{
+    Vector<Column> columns_for_selection;
```

Comment:
You forgot to use references here too.

Basically, try to never copy `Column` unless you need to.

Reply:
@awesomekling reminds me C++ doesn't allow `Vector<Column&>`; so maybe this is fine as is.

Reply:
Yeah, I think this is okay as-is. Unfortunately the branch has some conflicts with master, @DAlperin would you mind updating it so I can merge your changes? :)

Reply:
Conflicts fixed!